ui/src/app/features/app-user-list/application-user-list-facade.ts

```diff
@@ -79,6 +79,9 @@ export class ApplicationUserListFacade implements IFacade {
             return applicationRole === Role.OWNER ? applicationRole : globalRole;
           }
         }
+        if (applicationRole === Role.OWNER) {
+          return applicationRole;
```

Comment:
what if global role USER and local role also USER. applicationRole will not be returned